test/e2e/providers/azure/azure.go

```diff
@@ -110,7 +110,7 @@ func (p *Provider) GenerateMachineSet(withIgnoreLabel bool, replicas int32, wind
 
 	// create new machine provider spec for deploying Windows node in the same Location and Zone as master-0
 	providerSpec, err := p.newAzureMachineProviderSpec(masterProviderSpec.Location, masterProviderSpec.Zone,
-		windowsServerVersion)
+		masterProviderSpec.VMSize, masterProviderSpec.PublicLoadBalancer, windowsServerVersion)
```

Comment:
should the logic look into the configuration of `masterProviderSpec.PublicLoadBalancer` to ensure the outbound access?

Reply:
That would fall under the OCP installers responsibilities

Reply:
true